pyfakefs/fake_pathlib.py

```diff
@@ -392,6 +397,13 @@ class _FakeWindowsFlavour(_FakeFlavour):
         )
         pathmod = ntpath
 
+        def __init__(self, *args, **kwargs):
+            super().__init__(*args, **kwargs)
+            # Because this class is used with pure Windows paths,
+            # the separators must be filesystem-independent.
+            self.sep = "\\"
+            self.altsep = "/"
```

Comment:
> This may break the behavior of tests that set another path separator

Actually no, if you do this only if running under POSIX, see above:
```py
        def __init__(self, *args, **kwargs):
            super().__init__(*args, **kwargs)
            # If this class is used with pure Windows paths under POSIX,
            # the separators must be filesystem-independent.
            if not self.filesystem.is_windows_fs:
                self.sep = "\\"
                self.altsep = "/"
```